### PR TITLE
new subgenerator to display information

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Use: `yo jhipster:information`

I change the order in the ISSUE_TEMPLATE to make the copy/paste easier

### CODE REVIEW

**Comment by deepu105** [comment]
@pascalgrimaud @jdubois may be it should be `yo jhipster:info` so its easier to use as most projects use info keyword. what say. since this is not releases we can still change?

**Comment by jdubois** [comment]
Yes let's change it

**Comment by deepu105** [comment]
btw great idea :+1:

**Comment by deepu105** [comment]
ok ill change now

**Comment by deepu105** [comment]
done

File: generators/information/index.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                    this.log('```');
+                }
+                done();
+            }.bind(this));
+        },
+
+        displayYorc: function () {
+            var done = this.async();
+            this.log('##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**\n');
+            shelljs.exec('cat .yo-rc.json', {silent:true}, function (err, stdout, stderr) {
```
Comment (deepu105) [inline generators/information/index.js]: I dont think it works in windows though :) but we can improve it later

Comment (jdubois) [inline generators/information/index.js]: No, good catch! It shouldn't be too hard to output the file in pure JS

**Comment by pascalgrimaud** [comment]
Yes agree, yo jhipster:info is better!!

File: generators/information/index.js (hunk 1 above)
Comment (pascalgrimaud) [inline generators/information/index.js]: Oh right, I didn't test with Windows :D

File: generators/information/index.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                    this.log('```\n');
+                }
+                done();
+            }.bind(this));
+        },
+
+        displayEntities: function () {
+            var done = this.async();
+            this.log('##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**\n');
+            shelljs.exec('for f in `ls .jhipster`; do echo ${f%} ; cat .jhipster/${f%} ; done', {silent:true}, function (err, stdout, stderr) {
```
Comment (deepu105) [inline generators/information/index.js]: this is failing in windows as well

**Pull request merged**